src/main/drivers/vtx_common.h

```diff
@@ -82,3 +86,4 @@ void vtxCommonSetPitmode(uint8_t onoff);
 bool vtxCommonGetBandChan(uint8_t *pBand, uint8_t *pChan);
 bool vtxCommonGetPowerIndex(uint8_t *pIndex);
 bool vtxCommonGetPitmode(uint8_t *pOnoff);
+bool vtxCommonGetDeviceCapability(vtxDeviceCapability_t *pVtxDeviceCapability);
```

Comment:
I should note this PR's goal is NOT to change the existing API used for the Tramp/SmartAudio.  I followed the existing pattern so that it at least consistent.

I'll create an issue with links to some of these comments to go on the tech-debt pile.